src/test/java/com/clouway/cuse/SearchEngineContractTest.java

```diff
@@ -1001,6 +1035,15 @@ private void store(Employee... employees) {
     }
   }
 
+  private void storeAllEmployee(List<Object> employees) {
```

Comment:
This can either be renamed to `storeAllObjects` / `storeAll`, or the argument type can be changed to `List<Employee>`.